### DISCUSSION TITLE
[Future idea] Funding grants

### DISCUSSION THREAD

**Discussion by Sytten** [FossifyOrg/General-Discussion#80] [General]
I feel like it will be very hard to reach the expected 5000$/month on sponsorships anytime soon. So I wanted to start the discussion around sustainability via grants. I am aware of Futo (https://futo.org/fellows/) that does OSS grants. Maybe other people have ideas, I think some European countries have grants for OSS?

**Comment by naveensingh** [comment]
That is true. Looking at other open-source projects (including SMT, see [this](https://github.com/sponsors/tibbi) and [this](https://www.patreon.com/tiborkaputa)), the donation rates are too low to reach such a goal even after years.

**Reply by Sytten** [reply]
I would suggest you reach out to futo, I am pretty sure they would be interested to support you.

**Comment by Mr-Bajs** [comment]
True, but I think being a little more agressive when asking for financial support is fine. Thunderbird did it successfully. But of course it's a fine balance. A monthly popup asking for a donation or a patreon/liberapay support and explaining the situation, don't think will annoy people if they can toggle it of if they want.

**Reply by Sytten** [reply]
Feature based bounties is also something that could work. The app Nave nzb360 does it well (it needs a native integration).
I think also going through Google play & pay is a must have to reach critical mass.

**Comment by Mr-Bajs** [comment]
Duckduckgo gives donations, not grants it seamingly to more established insitutions but still could be something to keep in mind if going that route. Though I assume that they only donate to registed non-profit organisations, that might be an issue for many donations.
https://spreadprivacy.com/2023-duckduckgo-charitable-donations/
https://duckduckgo.com/donations

**Comment by Sytten** [comment]
I saw the nlnet foundation (https://nlnet.nl/) when I was looking at some rust stuff they support. The fossify android apps are not internet related per say, but one could argue they allow users to reclaim their privacy. Maybe something to ask them if one of their grant could fit?

**Reply by niccokunzmann** [reply]
Please have a look at the [NLNet foundation's funding](https://nlnet.nl/news/2024/20240201-call.html), especially [NGI Mobifree - More ethical and human mobile software](https://nlnet.nl/mobifree/) Deadline is 1st April. That could get you across a few months. They also mentor with networking, funding and other important FOSS topics, so please give it a go. Thanks for keeping Fossify going :heart:

**Reply by Sytten** [reply]
@naveensingh

**Reply by naveensingh** [reply]
Thanks for the suggestions. I'm a little busy now so I'll try these options later when I have some more time (probably after I'm done releasing the remaining apps).

**Reply by PanderMusubi** [reply]
See also:
- https://nlnet.nl/propose/
- https://nlnet.nl/funding.html

**Comment by Miriam-cpu** [comment]
Hi everyone, my name is Miriam, I am with the Institute for Technology and Journalism, an NGO based in Germany, Berlin. We run the website www.mobilsicher.de, where we offer easy accessible advice on privacy and safety for smartphones in German language. We have been recommending simple mobile tools for a long time and will now switch recommending your fork. We would also like to add a link to your donation website. But our readership will drop out the moment they see this nerdy git-subpage in english. It might work much better, if you offered these links on your webpage, possibly in German language? I am happy to help with a german version of your site. Also, it would be very helpful to be incorporated in some way. At least for German donors this is important - and its also easier to apply for grants. If there is interest in incorporating fossify in Germany I am happy to help - I have founded an NGO before. I am really grateful for your effort and would like to support it. I can also recommend NLnet for a starter

**Reply by ronniedroid** [reply]
Having the links directly inside the website is a good idea, I'll do that next week.

Also, I have made the website very easy to translate, just one `strings.json` file to be translated. And you can start translating as soon as https://github.com/FossifyOrg/fossifyorg.github.io/pull/6 gits merged. (or right now if you just copy the file from my fork :-) )

**Comment by naveensingh** [comment]
**Update:** 

My application to NLNet foundation was rejected in the final round. 

Since then, I have changed my mind about pursuing grants or funding from other organizations. I believe projects like Fossify should be funded by the community/users through donations and/or app sales on Google Play Store.

Thanks!
